bin/cli.js

```diff
@@ -149,6 +151,17 @@ const setRequestHeaders = () => {
 };
 
 const loadConfigMiddleware = async options => {
+  const validateConfig = () => {
+    const shouldValidate = options._.every(
```

Comment:
I like your idea here, but I think this might trigger unintentionally.

For example, what if we had a `hs project delete` command. We'd want to run validation on it, but this would be checking for:
```js
['project', 'delete'].every(item => !['init'...'delete'].includes(item))
```
That would return false because of `delete`. Same thing for `hs project list` and `hs accounts use`.

I think we'll need to be checking for the full command structure to make sure we're not unintentionally skipping config validation for certain commands. A map might be useful here. Maybe something like this?

```js
const NO_CONFIG_VALIDATION = {
  init: { skip: true },
  account: {
    skip: false
    use: { skip: true }
  }
}
```

^ That might not work well once fleshed out, but maybe it helps w/ brainstorming.

Another question I have about this is how it would handle command aliases. Like, if we didn't want to validate config for `hs account use`, but the user runs it like `hs accounts use`.

Reply:
Since we've eliminated all commands from the `NO_CONFIG_VALIDATION` apart from `init` and `auth`, I've gone with this structure:

```
const NO_CONFIG_VALIDATION = {
  init: { skip: true },
  auth: { skip: true },
};
``` 

We can revisit this later, if we need to add more commands. I'm going to resolve for now, but feel free to reopen!